src/types.ts

```diff
@@ -1,18 +1,31 @@
-import * as express from 'express';
 import * as http from 'http';
 import * as httpProxy from 'http-proxy';
 import * as net from 'net';
 
-export interface Request extends express.Request {}
-export interface Response extends express.Response {}
+export interface Request extends http.IncomingMessage {
+  originalUrl?: string; // in case express, connect
+  hostname?: string; // in case express
+  host?: string; // in case express
+}
+export interface Response extends http.ServerResponse {}
 
-export interface RequestHandler extends express.RequestHandler {
+export interface RequestHandler<
+  Request extends http.IncomingMessage = http.IncomingMessage,
```

Comment:
Making type parameters in all exported types optional to save some possibly unnecessary breaking errors like `RequestHandler requires two parameters none passed`. More [here](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/48591#discussion_r502801531).